va/va_dec_vp9.h

```diff
@@ -189,8 +189,22 @@ typedef struct  _VADecPictureParameterBufferVP9
      */
     uint8_t                 bit_depth;
 
+    /** \brief loop filter scemantics
+     */
+    uint8_t                 mode_ref_delta_enabled;
+    uint8_t                 mode_ref_delta_update;
+
+    /** \brief Quantization params
+     */
+    uint8_t                 base_qindex;
+    int8_t                  y_dc_delta_q;
+    int8_t                  uv_ac_delta_q;
+    int8_t                  uv_dc_delta_q;
```

Comment:
What are you going to use the top-level quant values for?  They don't do anything in themselves because the segment features can modify them, and the derived per-segment values are present in `VASegmentParameterVP9`.

Reply:
In case, the segmentation is not enabled, the top level value is needed.  We saw failures in playback without these values.

Reply:
Can you explain the details of the case you are having problems with?  When segmentation is not enabled all `segment_id`s are zero so all mode info blocks use the `quant_scale` values supplied for segment zero.

Reply:
The patch doesn't make sense to me either. It is not mandatory to send the uncompressed header to the driver which means there wouldn't be a way to parse these fields inside the UMD.
Also, please note that the SegParam[0] will have all the required values even if segmentation is disabled.